solidity/contracts/test/TestRecipient.sol

```diff
@@ -42,4 +41,8 @@ contract TestRecipient is
         lastCaller = msg.sender;
         lastCallMessage = message;
     }
+
+    function setInterchainSecurityModule(address _ism) external onlyOwner {
```

Comment:
Do you need to initialize or does it happen with the constructor?

Reply:
Just defaults to `address(0)` which means the default ISM is used